app/src/main/java/pub/devrel/easypermissions/sample/FeatureComponent.java

```diff
@@ -0,0 +1,58 @@
+package pub.devrel.easypermissions.sample;
```

Comment:
This was very instructive to me in understanding your proposal, but could you actually revert all the changes to the same and just include the library changes?

I'll probably document this in the `README`, since it's for advanced users they'll likely have their own usage patterns in mind anyway.